packages/trace-viewer/src/ui/actionList.tsx

```diff
@@ -128,8 +133,9 @@ export const renderAction = (
       {action.method === 'goto' && action.params.url && <div className='action-url' title={action.params.url}>{action.params.url}</div>}
       {action.class === 'APIRequestContext' && action.params.url && <div className='action-url' title={action.params.url}>{excludeOrigin(action.params.url)}</div>}
     </div>
-    {(showDuration || showBadges) && <div className='spacer'></div>}
+    {(showDuration || showBadges || showAttachments) && <div className='spacer'></div>}
     {showDuration && <div className='action-duration'>{time || <span className='codicon codicon-loading'></span>}</div>}
+    {showAttachments && <ToolbarButton icon='attach' title='Open Attachment' onClick={revealAttachments} />}
```

Comment:
Perhaps put this to the left of the duration? Misaligned durations look ugly.

Reply:
addressed: <img width="1392" alt="Screenshot 2024-10-24 at 14 33 00" src="https://github.com/user-attachments/assets/91624306-14e8-4406-a1b3-01fa0e822e03">